apps/actions-dependencies-updater/README.md

```diff
@@ -0,0 +1,61 @@
+# Actions Dependencies Updater
+
+This tool has two main purposes:
+
+1. Checking if your workflows rely (directly or transitively) on any node12 or
+   node16 actions.
+2. Updating all actions within a repo's workflows to the newest tag/release for
+   that action.
+
+## Setup
+
+Have [`pnpm`](https://pnpm.io/) and [`direnv`](https://direnv.net/) setup.
+
+1. Install Dependencies
+
+```bash
+pnpm install
+```
+
+2. Populate a `.envrc` file within this project's directory
+
+```bash
+# Requires "repo" access, and authorized to read the repo(s) you are testing.
+export GH_ACCESS_TOKEN="<Github PAT>"
+```
+
+3.  `direnv allow`
```

Comment:
If we use https://www.npmjs.com/package/dotenv then we dont have to ask the user to have another system level package installed

Reply:
Good point